components/molecules/PendingBenefitDetails.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import VerticalStepper from '../atoms/VerticalStepper'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+
+const PendingBenefitCard = (props) => {
+  const t = props.locale === 'en' ? en : fr
+
+  return (
+    <div className="grid grid-cols-1 divide-y divide-grey-500 font-display my-4">
+      <section>
+        <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-7 sm:gap-4 sm:mx-auto sm:mt-4">
+          <div
+            id="claimUpdates"
+            className="col-span-2 sm:col-span-3 lg:col-span-2 sm:w-60 ml-7 mb-6 sm:mb-3"
```

Comment:
I think this broke the mobile work that was completed as part of PR #181 

![image](https://user-images.githubusercontent.com/14844029/151877411-fd60ffe2-47d5-4188-b7aa-acb4bfbd7b07.png)

Reply:
Do you mean the Left Margin?
I just changed the name of this file. I worked on Active card.

Reply:
This file is different than its previous `PendingBenefitCard.jsx` something changed somewhere.